ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -271,21 +291,30 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         self.animatesWhenPressed = animatesWhenPressed
         self.accessoryType = accessoryType
 
-        titleButtonLeadingImageView.image = titleImage
-        titleButtonLeadingImageView.isHidden = titleImage == nil
+        titleLeadingImageView.image = titleImage
+        titleLeadingImageView.isHidden = titleImage == nil
+
+        setupTitleLine(titleContainer, label: titleLabel, trailingImageView: titleTrailingImageView, text: title, interactive: interactivePart.contains(.title), accessoryType: accessoryType)
+        if titleLeadingImageView.image != nil {
+            titleContainer.insertArrangedSubview(titleLeadingImageView, at: 0)
+        }
 
-        setupButton(titleButton, label: titleButtonLabel, trailingImageView: titleButtonTrailingImageView, text: title, interactive: interactivePart.contains(.title), accessoryType: accessoryType)
         // Check for strict equality for the subtitle button's interactivity.
-        // If the whole area is active, we'll stretch the title button to adjust the hit area
-        // while still only keeping one button active from an accessibility standpoint.
-        setupButton(subtitleButton, label: subtitleButtonLabel, trailingImageView: subtitleButtonImageView, text: subtitle, interactive: interactivePart == .subtitle, accessoryType: accessoryType)
+        // If the whole area is active, we'll use the title as the main accessibility item.
+        setupTitleLine(subtitleContainer, label: subtitleLabel, trailingImageView: subtitleImageView, text: subtitle, interactive: interactivePart == .subtitle, accessoryType: accessoryType)
 
-        let subtitleIsNilOrEmpty = subtitle?.isEmpty ?? true
         minimumContentSizeCategory = .large
-        maximumContentSizeCategory = subtitleIsNilOrEmpty ? .extraExtraLarge : .large
 
-        invalidateIntrinsicContentSize()
-        setNeedsLayout()
+        containingStackView.removeAllSubviews()
+        containingStackView.alignment = alignment.stackViewAlignment
+        containingStackView.addArrangedSubview(titleContainer)
+
+        if subtitle?.isEmpty == false {
+            maximumContentSizeCategory = .large
+            containingStackView.addArrangedSubview(subtitleContainer)
```

Comment:
Couldn't you just hide the `subtitleContainer` instead of removing all the subviews and then adding them back?